diff --git a/node_modules/redux/types/types/store.d.ts b/node_modules/redux/types/types/store.d.ts
index 3eb436f..bff9739 100644
--- a/node_modules/redux/types/types/store.d.ts
+++ b/node_modules/redux/types/types/store.d.ts
@@ -24,7 +24,7 @@ declare const $CombinedState: unique symbol;
 interface EmptyObject {
     readonly [$CombinedState]?: undefined;
 }
-export type CombinedState<S> = EmptyObject & S;
+export type CombinedState<S> = S;
 /**
  * Recursively makes combined state objects partial. Only combined state _root
  * objects_ (i.e. the generated higher level object with keys mapping to
